unfetter-discover/unfetter-ui
# Issue 1032 (Dynamic step generation in baseline wizard)

Fixes unfetter-discover/unfetter#1032

As capability groups and capabilities are selected in the baselines wizard, steps are dynamically added to the sidebar of the wizard, and the user can click `Continue` and `Back` to iterate through them.  Deletion of a capability group in the first step will also delete any associated capabilities from the baseline and the sidebar will update.

## To test:
1. Pull this PR
2. Restart unfetter-ui:  `docker-compose restart unfetter-ui`
3. Access baselines wizard through Create in the baselines list
4. Iterate through the wizard, selecting capability groups and capabilities
5. Note update of step tree in sidebar
6. Click Continue and Back and make sure nodes expand and are highlighted as expected
7. Remove a group and ensure it is also removed from the sidebar, and that its associated capabilities can now be selected
